specs/signature-specification.md

```diff
@@ -176,7 +176,7 @@ See [Guidelines for implementations of the Notary Project signature specificatio
 These attributes are considered unsigned with respect to the signing key that generates the signature.
 
 - **Certificate Chain**: This is a REQUIRED attribute that contains the ordered list of X.509 public certificates associated with the signing key used to generate the signature. The ordered list starts with the signing certificate, any intermediate certificates and ends with the root certificate. The certificate chain MUST be authenticated against a trust store as part of signature validation. Specific requirements for the certificates in the chain are provided [here](#certificate-requirements).
-- **Timestamp signature** : An OPTIONAL counter signature which provides [authentic timestamp](#signing-time)e.g. Time Stamp Authority (TSA) generated timestamp signature. Only [RFC3161][ietf-rfc3161] compliant TimeStampToken are currently supported.
+- **Timestamp Signature** (critical): An OPTIONAL countersignature which provides [authentic signing time](#signing-time--authentic-signing-time) for signing scheme [`notary.x509`](./signing-scheme.md/#notaryx509), e.g. Timestamp Authority (TSA) generated timestamp signature. Only [RFC3161][ietf-rfc3161] compliant TimeStampToken are currently supported. When provided, this claim is only validated and used under signing scheme [`notary.x509`](./signing-scheme.md/#notaryx509).
```

Comment:
It is indeed "authentic timestamp", not "authentic signing time".

Reply:
Updated after discussion.